vm/2.0/modules/ROOT/pages/index.adoc

```diff
@@ -3,75 +3,57 @@
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-VM Connector v2.0
+Anypoint Connector for Virtual Machine (VM Connector) handles intra-app and inter-app communication through asynchronous queues that can be transient or persistent.
 
-Anypoint Connector for Virtual Machine (VM) (VM Connector) handles intra-app and inter-app communication through asynchronous queues that can be transient or persistent.
+* Transient queues are faster than persistent queues, but they are not reliable in the case of a system crash
+* Persistent queues are slower but reliable
 
-* Transient queues are faster than persistent queues, but they are not reliable in the case of a system crash.
-* Persistent queues are slower but reliable.
+When running on a single instance, persistent queues work by serializing and storing the contents on the disk. However, when running in cluster mode, the memory grid backs the persistent queues. Thereby, when a flow uses the VM Connector to publish content to a queue, Mule runtime engine determines whether to process the message in the same origin node or to send the message to the cluster to pick another node. The VM Connector facilitates the load distribution across a cluster.
+In either case, when using persistent queues the contents you send must be serializable. Even though you can enable Kryo serialization to get a wider range of serializable values, Kryo does have some limitations. Therefore, try to:
```

Comment:
```suggestion
In either single-instance or cluster-mode processing, when using persistent queues, the data you send must be serializable. Although you can enable Kryo serialization to get a wider range of serializable values, Kryo does have some limitations. Therefore, try to:
```
Question about "Therefore, try to:"
Are we saying that you must try to do these things to address the Kyro limitations, or are we saying that you should try to do these things INSTEAD of using Kyro?
And do we need to specify what those limitations are?

Reply:
GLOBALLY review how this topic variously refers to the information in the communication: contents? data? values? message?
Can we standardize to fewer or just one term?